src/IPC/PipeTx.cc

```diff
@@ -68,7 +70,10 @@ std::string PipeTx::pipeTx(void) {
             #if PRINT
             std::cout << "Last string: " << readbuf << std::endl;
             #endif
-            write(this->fd, this->readbuf, remaining_bytes);
+            write_bytes = write(this->fd, this->readbuf, remaining_bytes);
+            if (write_bytes == -1) {
+                throw std::runtime_error("Pipe-Tx write failed");
+            }
             usleep(100000); // 100ms
```

Comment:
What is the purpose of those 'usleep' calls ? Here, and in 'fileSizeTx'

Reply:
I will try to remove these in the next iteration of pipe :+1: